modules/stanford_events_importer/src/Plugin/Field/FieldWidget/StanfordEventsImporterAPIURLFieldWidget.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+namespace Drupal\stanford_events_importer\Plugin\Field\FieldWidget;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\link\LinkItemInterface;
+use Drupal\Component\Utility\UrlHelper;
+use Drupal\link\Plugin\Field\FieldWidget\LinkWidget;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+
+/**
+ * Plugin implementation of the 'stanford_events_importer_apiurl_field_widget'.
+ *
+ * @FieldWidget(
+ *   id = "stanford_events_importer_apiurl_field_widget",
+ *   module = "stanford_events_importer",
+ *   label = @Translation("Stanford Events API URL Builder Widget"),
+ *   field_types = {
+ *     "link"
+ *   }
+ * )
+ */
+class StanfordEventsImporterAPIURLFieldWidget extends LinkWidget {
+
+  /**
+   * Path to feed xml.
+   */
+  const XML_FEED = "https://events.stanford.edu/xml/drupal/v2.php";
+
+  /**
+   * {@inheritdoc}
+   */
+  public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
+
+    // De-falt.
+    $element = parent::formElement($items, $delta, $element, $form, $form_state);
+    $element['#attached']['library'][] = 'core/drupal.states';
```

Comment:
Is this needed? i thought when you add the `#states` it gets added already.

Reply:
I'm having issues with this and the `add another` Currently debugging.